format_values.md

```diff
@@ -4,17 +4,15 @@ This document is a list of standardized values that can be used in certain field
 
 ## Frameworks
 
-The following is a list of standardized Framework names.
+The following list contains standardized Framework names which SHOULD be used as the respective projects ID.
```

Comment:
I think this is a useless requirement if it's a SHOULD.

Reply:
A `SHOULD` places no artificial constraints on spec writers, though signalling that this is the preferred way of referencing these frameworks. Since no implementation can effectively enforce this name anyway (because there is no way of testing that a modip spec actually contains a framework or just some random mod), it is better to use `SHOULD` than, for example, `MUST`.

Reply:
Agreed, I would change the wording to something like "if you are using one of these frameworks listed below, you MUST use these IDs. This only applies to those frameworks. For frameworks not listed below, there is no formal requirement of what IDs you must use."

Reply:
Nevermind, disregard earlier message. I think this is good.

Reply:
For installing a modpack: it doesn't matter what the IDs are, right? For decentralized metadata files, you use whatever the official ID is. And for a centralized repository, you use whatever the repository says. That's why I don't see the value of the requirement.